fMRIVolume/scripts/DistortionCorrectionAndEPIToT1wReg_FLIRTBBRAndFreeSurferBBRbased.sh

```diff
@@ -409,15 +409,17 @@ case $DistortionCorrection in
         Files="PhaseOne_gdc_dc PhaseTwo_gdc_dc SBRef_dc"
         for File in ${Files}
         do
+            ReferenceImage=${SubjectFolder}/T1w/T1w_acpc_dc.nii.gz
```

Comment:
`ReferenceImage` is independent of the `for File in ${Files}` loop, so can be defined above that.

Reply:
Moved outside of the loop.